vita3k/config/src/config.cpp

```diff
@@ -276,4 +277,27 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
     return Success;
 }
 
+bool is_tracy_advanced_profiling_active_for_module(std::vector<std::string> &active_modules, std::string module, int *index) {
```

Comment:
Should I warp this one in a `#ifdef TRACY_ENABLE ... #endif` check as well so it won't get compiled on Release builds or should I leave it as it is right now? The only code that executes it is in `vita3k/gui/src/settings_dialog.cpp` and that code is already set not to be compiled on Release builds.